src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,21 @@
+<!-- Navbar -->
+    <div th:fragment ="navbar">
+            <nav class="navbar navbar-expand-lg bg-warning navbar-dark py-1">
+            <div class="container-fluid">
+                <div class="col-md-1 text-center d-none d-md-block">
+                    <img class="img-fluid" src="/static/img/doctors_symbol.png" />
```

Comment:
Looks like this needs to be changed to `/img/doctors_symbol.png`, the logo is missing in the JUnit screenshots: https://github.com/CS4850Group5A/VirtuDoc/suites/5466804591/artifacts/174067901

Reply:
![1645997296570-_notifications](https://user-images.githubusercontent.com/4678601/155917170-9306c368-b212-4505-a423-40010c2c31fd.png)

Reply:
I must have an out-of-date branch, because img src = /img/doctors_symbol.png shows that it is not the correct directory, HOWEVER, as soon as recompile with docker, I see Records/Video/Message/Home all have the logo back. However, Appointment does not. Very odd. So, pushing the fix regardless and re-updating branch.
![Screenshot (69)](https://user-images.githubusercontent.com/75335524/155920746-4371c487-f663-4f0b-b869-9b839d4234c5.png)